agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/HttpServerTracer.java

```diff
@@ -195,6 +201,44 @@ protected void onRequest(final Span span, final REQUEST request) {
     // TODO set resource name from URL.
   }
 
+  protected void onConnectionAndRequest(Span span, CONNECTION connection, REQUEST request) {
+    String flavor = flavor(connection, request);
+    if (flavor != null) {
+      SemanticAttributes.HTTP_FLAVOR.set(span, flavor);
+    }
+    SemanticAttributes.HTTP_CLIENT_IP.set(span, clientIP(connection, request));
+  }
+
+  protected String clientIP(CONNECTION connection, REQUEST request) {
+    // try Forwarded
```

Comment:
do you think we need to support this header?

Reply:
yeah, mozilla says it's the real standard while x-forwarded-for is the defacto standard. best to get both imo. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded